network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_card.html

```diff
@@ -24,6 +24,10 @@
   {% endif %}
 {% endblock %}
 
+{% block description %}
+  <p class="tw-body tw-line-clamp-3"> {{page.specific.localized.get_meta_description}} </p>
```

Comment:
**Suggestion** I think we should remove this whitespace inside the tag and rather add it in the braces. 

```suggestion
  <p class="tw-body tw-line-clamp-3">{{ page.specific.localized.get_meta_description }}</p>
```

Keeps it readable but avoids rendering issues.